### ISSUE TITLE
CLI Commands: migrations not added to relative-up output directories

### ISSUE THREAD

**Issue by natemcmaster** [aspnet/EntityFramework#5187]
```
C:\sln\projectA> dotnet ef migrations add Initial --output-dir ..\elsewhere
C:\sln\projectA> dir ..\elsewhere
0 files

C:\sln\projectA> dir .\elsewhere
xxxx_Initial.cs
ContextSnapshot.cs
```

**Comment by divega** [comment]
Do you know why?

**Comment by natemcmaster** [comment]
Yes. We use `outputDir` to calculate the namespace of the migrations. When we write files, we pick the folder based on the namespace, not the outputDir. It’s probably a simple fix.

**Comment by divega** [comment]
Got it. We should achieve better separation of namespace from output dir in general (even if we still use output dir to calculate the namespace by default) because of https://github.com/aspnet/EntityFramework/issues/4524 anyway.

**Comment by natemcmaster** [comment]
Fixed with #5263
